### ISSUE TITLE
Use BossBar to show the warmup's progress of siege machines

### ISSUE THREAD

**Issue by Jose-Paulo-FaD** [Multitallented/Civs#31]
Just a quick suggestion after seeing the the War Guide video. I think BossBars are a pretty neat and "immersive" way to show progress information.

**Comment by Multitallented** [comment]
Good idea. I'll have to think of when exactly to use them, but I love the idea.
